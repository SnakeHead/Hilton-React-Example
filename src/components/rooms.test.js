import React from 'react';
import Rooms from './rooms';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Rooms/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
