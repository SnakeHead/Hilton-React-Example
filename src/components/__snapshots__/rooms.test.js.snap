// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="roomPanel"
  >
    <span
      className="roomTitle"
    >
      Room 
      1
    </span>
    <div
      className="selectorPanel"
    >
      <form>
        <div
          className="roomSelect"
        >
          <label
            htmlFor="roomAdult"
          >
            Adults
            <br />
            (18+)
          </label>
          <select
            aria-label="roomAdults"
            disabled={false}
            name="roomAdult"
            onChange={[Function]}
            value={0}
          >
            <option>
              1
            </option>
            <option>
              2
            </option>
          </select>
        </div>
        <div
          className="roomSelect"
        >
          <label
            htmlFor="roomChild"
          >
            Children
            <br />
            (0-17)
          </label>
          <select
            aria-label="roomChildren"
            disabled={false}
            name="roomChild"
            onChange={[Function]}
            value={0}
          >
            <option>
              0
            </option>
            <option>
              1
            </option>
            <option>
              2
            </option>
          </select>
        </div>
      </form>
    </div>
  </div>
  <div
    className="roomPanel"
  >
    <span>
      <input
        aria-label="roomCheckbox"
        checked={false}
        name="checkbox"
        onChange={[Function]}
        type="checkbox"
      />
    </span>
    <span
      className="roomTitle"
    >
      Room 
      2
    </span>
    <div
      className="selectorPanel"
    >
      <form>
        <div
          className="roomSelect"
        >
          <label
            htmlFor="roomAdult"
          >
            Adults
            <br />
            (18+)
          </label>
          <select
            aria-label="roomAdults"
            disabled={true}
            name="roomAdult"
            onChange={[Function]}
            value={0}
          >
            <option>
              1
            </option>
            <option>
              2
            </option>
          </select>
        </div>
        <div
          className="roomSelect"
        >
          <label
            htmlFor="roomChild"
          >
            Children
            <br />
            (0-17)
          </label>
          <select
            aria-label="roomChildren"
            disabled={true}
            name="roomChild"
            onChange={[Function]}
            value={0}
          >
            <option>
              0
            </option>
            <option>
              1
            </option>
            <option>
              2
            </option>
          </select>
        </div>
      </form>
    </div>
  </div>
  <div
    className="roomPanel"
  >
    <span>
      <input
        aria-label="roomCheckbox"
        checked={false}
        name="checkbox"
        onChange={[Function]}
        type="checkbox"
      />
    </span>
    <span
      className="roomTitle"
    >
      Room 
      3
    </span>
    <div
      className="selectorPanel"
    >
      <form>
        <div
          className="roomSelect"
        >
          <label
            htmlFor="roomAdult"
          >
            Adults
            <br />
            (18+)
          </label>
          <select
            aria-label="roomAdults"
            disabled={true}
            name="roomAdult"
            onChange={[Function]}
            value={0}
          >
            <option>
              1
            </option>
            <option>
              2
            </option>
          </select>
        </div>
        <div
          className="roomSelect"
        >
          <label
            htmlFor="roomChild"
          >
            Children
            <br />
            (0-17)
          </label>
          <select
            aria-label="roomChildren"
            disabled={true}
            name="roomChild"
            onChange={[Function]}
            value={0}
          >
            <option>
              0
            </option>
            <option>
              1
            </option>
            <option>
              2
            </option>
          </select>
        </div>
      </form>
    </div>
  </div>
  <div
    className="roomPanel"
  >
    <span>
      <input
        aria-label="roomCheckbox"
        checked={false}
        name="checkbox"
        onChange={[Function]}
        type="checkbox"
      />
    </span>
    <span
      className="roomTitle"
    >
      Room 
      4
    </span>
    <div
      className="selectorPanel"
    >
      <form>
        <div
          className="roomSelect"
        >
          <label
            htmlFor="roomAdult"
          >
            Adults
            <br />
            (18+)
          </label>
          <select
            aria-label="roomAdults"
            disabled={true}
            name="roomAdult"
            onChange={[Function]}
            value={0}
          >
            <option>
              1
            </option>
            <option>
              2
            </option>
          </select>
        </div>
        <div
          className="roomSelect"
        >
          <label
            htmlFor="roomChild"
          >
            Children
            <br />
            (0-17)
          </label>
          <select
            aria-label="roomChildren"
            disabled={true}
            name="roomChild"
            onChange={[Function]}
            value={0}
          >
            <option>
              0
            </option>
            <option>
              1
            </option>
            <option>
              2
            </option>
          </select>
        </div>
      </form>
    </div>
  </div>
  <button
    className="submitButton"
    onClick={[Function]}
  >
    Submit
  </button>
</div>
`;
